quickwit/quickwit-rest-client/src/rest_client.rs

```diff
@@ -31,13 +33,14 @@ use serde::Serialize;
 use serde_json::json;
 use tokio::fs::File;
 use tokio::io::{AsyncBufReadExt, AsyncRead, BufReader};
+use tokio_stream::wrappers::LinesStream;
 
 use crate::error::Error;
 use crate::models::{ApiResponse, IngestSource};
 
 pub static DEFAULT_ADDRESS: &str = "http://127.0.0.1:7280";
 pub static DEFAULT_CONTENT_TYPE: &str = "application/json";
-pub static INGEST_CHUNK_SIZE_IN_BYTES: usize = 4 * 1024 * 1024;
+pub static INGEST_CHUNK_SIZE_IN_BYTES: usize = 9 * 1024 * 1024; // 9MiB
```

Comment:
Why not 10?

Reply:
10 MiB is a bit above what's allowed which is specified in 10MB (10 * 1000 * 1000) as opposed to (9 * 1024 * 1024)
We can harmonize to MB (10 * 1000 * 1000) because this is more readable.

Reply:
Let's harmonize.